coffee-chats/src/main/java/com/google/step/coffee/Nullable.java

```diff
@@ -0,0 +1,3 @@
+package com.google.step.coffee;
+
+public @interface Nullable {}
```

Comment:
I believe this annotation is already built into javax.annotation is it not? I am not sure what the reasoning here is to recreate it

Reply:
It doesn't unless we want to add another dependency to pom.xml